backend/vaa-strapi/src/functions/utils/appCustomization.ts

```diff
@@ -0,0 +1,61 @@
+import {staticSettings} from 'vaa-shared';
+import en from './translations/en/dynamic.json';
```

Comment:
The same translation files are inelegantly copied from the frontend until we can move them to a shared module. The global e2e tests check, however, that the files match.

Reply:
Yeah, see #608. Of course, if there's a suitable update to Svelte 4, then we can anticipate this.